File: C#/Essencial/Estrutura_PARA.cs
using System;

namespace Programa
{
    class Program
    {
        static void Main(string[] args)
        {
            int n, i, x, soma;

            Console.Write("Quantos números serão digitados? ");
            n = int.Parse(Console.ReadLine());

            soma = 0;
            for(i = 1; i <= n; i++)
            {
                Console.Write("Digite um número: ");
                x = int.Parse(Console.ReadLine());
                soma = soma + x;
            }

            Console.WriteLine("Soma" + soma);
        }
    }
}

File: Java/Inicio/src/Essencial/Estrutura_para.java
package Essencial;

import java.util.Scanner;

public class Estrutura_para {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int N, i, soma, x;
        System.out.println("Quantos números serão digitados: ");
        N = sc.nextInt();

        soma = 0;
        for(i = 1; i <= N; i++){
            System.out.println("Digite um número: ");
            x = sc.nextInt();
            soma += x;
        }

        System.out.println("SOMA = " + soma);
        sc.close();
    }
}
